RELEASE.md

```diff
@@ -83,3 +86,56 @@ The usual steps are:
     * Copy the bundle files from `github.com/kuadrant/kuadrant-operator/tree/v0.W.Z/bundle`
     * Copy `github.com/kuadrant/kuadrant-operator/tree/v0.W.Z/bundle.Dockerfile` with the proper fix to the COPY commands
       (i.e. remove /bundle from the paths)
+
+--- 
+
+## Release file format.
+
+This example of the `release.toml` file uses tag `v1.0.1` as reference.
+
+```toml
```

Comment:
The format LGTM. What tools could we leverage from GHActions (bash) or Makefile to read those?